components/TextFields/src/MDCTextInputControllerFullWidth.m

```diff
@@ -424,6 +424,22 @@ - (void)setCharacterCountViewMode:(UITextFieldViewMode)characterCountViewMode {
   }
 }
 
+- (void)setDisabledColor:(UIColor *)disabledColor {
```

Comment:
I added some unused warnings to the project and I did get a pile of unused parameter complaint from the compiler.

```
-Wunused-comparison -Wunused-const-variable -Wunused-exception-parameter -Wunused-function -Wunused-label -Wunused-member-function -Wunused-parameter -Wunused-private-field -Wunused-property-ivar -Wunused-result -Wunused-value -Wunused-variable -Wunused-volatile-lvalue -Wused-but-marked-unused
```